index.html

```diff
@@ -0,0 +1,453 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>aaaaaaaaaaaaaaaaaaa</title>
+  </head>
+  <body>
+    <header>
```

Comment:
The color of the header on Mobile screen, according to the Figma design is
![image](https://user-images.githubusercontent.com/61361037/210972008-3b287cfd-a2ff-4f75-a1a5-3c328cec76c9.png)
Kindly update the color of the header according to the design provided.